component/src/main/wso2/kafka/consumer.bal

```diff
@@ -99,12 +101,12 @@ public type ConsumerConfig record {
 
 # Type related to consumer record.
 #
-# + key - key that is included in the record
-# + value - record content
-# + offset - offset value
-# + partition - partition to which the record is stored
-# + timestamp - timestamp of the record, in milliseconds since epoch
-# + topic - topic of the record
+# + key - Key that is included in the record.
```

Comment:
Again, need to be consistent. Here, the sentences are finished with a period while in the doc block above, that is not the case